tests/testthat/_snaps/dots-ellipsis.md

```diff
@@ -23,19 +22,78 @@
     Output
       <error/rlib_error_dots_nonempty>
       Error in `f()`:
-      `...` is not empty.
-      i These dots only exist to allow future extensions and should be empty.
-      x We detected these problematic arguments:
-      * `xy`
-      i Did you misspecify an argument?
+      `...` must be empty.
+      x Problematic argument:
+      * xy = 4
     Code
       (expect_error(f0(xy = 4), class = "rlib_error_dots_nonempty"))
     Output
       <error/rlib_error_dots_nonempty>
       Error in `f0()`:
-      `...` is not empty.
-      i These dots only exist to allow future extensions and should be empty.
-      x We detected these problematic arguments:
-      * `xy`
-      i Did you misspecify an argument?
+      `...` must be empty.
+      x Problematic argument:
+      * xy = 4
+
+# expression contents are mentioned
+
+    Code
+      f("foo")
+    Error <rlib_error_dots_nonempty>
```

Comment:
Are you deliberately using the old testthat style?

Reply:
I can't use the new style because rlang isn't in its own Imports, so I'll wait for the new style to become the default.